homeassistant/components/binary_sensor/ping.py

```diff
@@ -126,7 +129,18 @@ def ping(self):
                     'avg': rtt_avg,
                     'max': rtt_max,
                     'mdev': ''}
-
+            if 'max/' not in str(out):
+                _LOGGER.debug("Detected Alpine ping...")
+                match = PING_MATCHER_ALPINE.search(str(out).split('\n')[-1])
+                rtt_min, rtt_avg, rtt_max = match.groups()
+                _LOGGER.debug(
```

Comment:
The `state_changed` event will be logged and contains pretty much the content of the debug output.